docker-compose.prod.override.yml

```diff
@@ -24,6 +24,11 @@ services:
       traefik.http.routers.tools4msp_geoplatform_geonode_router.rule: Host(`dev.geoplatform.tools4msp.eu`)
       traefik.http.routers.tools4msp_geoplatform_geonode_router.tls.certresolver: le
       traefik.http.routers.tools4msp_geoplatform_geonode_router.entrypoints: websecure
+    volumes:
+      - type: bind
+        source: $PWD/geonode.conf.envsubst
```

Comment:
$PWD? Wouldn't ./ be enough?

Reply:
I agree, using `./` is likely better, as the path is taken relative to the compose file, rather than whatever the current directory may be. I'll make the change